### PR TITLE
Complete BLL

### PR DESCRIPTION
**Pull request by UNDogMan**

### CODE REVIEW

File: WebAutopark.BusinessLogic/Services/OrderService.cs [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+        public OrderService(IRepository<Order> repository, IMapper mapper)
+        {
+            this.repository = repository;
+            this.mapper = mapper;
+        }
+
+        public Task Create(OrderDTO item)
+        {
+            return repository.Create(mapper.Map<Order>(item)); ;
```
Comment (PushMeNow) [inline WebAutopark.BusinessLogic/Services/OrderService.cs:27]: Excess colon

File: WebAutopark.BusinessLogic/Services/VehicleService.cs [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+            return vehicleRepository.Update(mapper.Map<Vehicle>(item));
+        }
+
+        public async Task<float> GetMaxmileage(VehicleDTO vehicle)
+        {
+            var type = await vehicleTypeRepository.Get(vehicle.VehicleTypeID);
+            return vehicle.Weight * WeightCoefficient + type.TaxCoefficient * BaseTax + MinTax;
+        }
+
+        public async Task<float> GetTaxPerMount(VehicleDTO vehicle)
```
Comment (PushMeNow) [inline WebAutopark.BusinessLogic/Services/VehicleService.cs:65]: I don't see async operations here. Let's move logic of the method in helper Calculator

File: WebAutopark.BusinessLogic/Services/VehicleService.cs [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+        }
+        public Task Update(VehicleDTO item)
+        {
+            return vehicleRepository.Update(mapper.Map<Vehicle>(item));
+        }
+
+        public async Task<float> GetMaxmileage(VehicleDTO vehicle)
+        {
+            var type = await vehicleTypeRepository.Get(vehicle.VehicleTypeID);
+            return vehicle.Weight * WeightCoefficient + type.TaxCoefficient * BaseTax + MinTax;
```
Comment (PushMeNow) [inline WebAutopark.BusinessLogic/Services/VehicleService.cs:62]: Let's move logic of this line in helper Calculator

File: WebAutopark.BusinessLogic/Services/VehicleService.cs [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+using WebAutopark.DataAccess.Repository.Base;
+using WebAutopark.DataAccess.Entities;
+
+namespace WebAutopark.BusinessLogic.Services
+{
+    public class VehicleService : IVehicleService
+    {
+        private const float WeightCoefficient = 0.0013f;
+        private const int BaseTax = 30;
+        private const int MinTax = 5;
```
Comment (PushMeNow) [inline WebAutopark.BusinessLogic/Services/VehicleService.cs:18]: I don't see the reason to create global constant here. We will not use them anywhere. Better create local constants.

**Review by PushMeNow — CHANGES_REQUESTED** [review]
Hi man!
I made some comments for you. Let me know if any questions.

**Review by dbogdan2000 — APPROVED** [review]

**Review by PushMeNow — APPROVED** [review]
Approved

**Pull request merged**